app/services/users.js

```diff
@@ -0,0 +1,24 @@
+const { inspect } = require('util');
+const bcrypt = require('bcrypt');
+
+const errors = require('../errors');
+const Users = require('../models/users');
+const logger = require('../logger');
+
+exports.createUser = newUser =>
+  Users.findOrCreate(newUser)
```

Comment:
Mmm maybe you are confused, because `findOrCreate`receive two params, first for the conditions to check if exist the user, and second the values if the user does not exist. I don't remember the order exactly, but please, check this